deployment.rst

```diff
@@ -48,6 +48,16 @@ via FTP/SCP (or similar method). This has its disadvantages as you lack control
 over the system as the upgrade progresses. This method also requires you
 to take some manual steps after transferring the files (see `Common Post-Deployment Tasks`_)
 
+Shared Hosted Server
+~~~~~~~~~~~~~~~~~~~~
+
+On a shared hosted server, the simplest way to deploy your Symfony application is
+using symfony/apache-pack (which creates an .htaccess inside public folder) and
```

Comment:
Nice addition i think it is usefull
but i would maybe add it in this part instead

```
E) Other Things!
There may be lots of other things that you need to do, depending on your setup:

```

Like: `If you are and a shared host plateform running Apache Web Server, you may install the symfony/apache-pack` as well

wdyt?